end-to-end-tests/resources/signup_resource.robot

```diff
@@ -16,7 +16,7 @@ Signup With Email
     Open Browser To Main Page
     Click get started button
     Insert Email    ${email}
-    Click Begin button
+    Click Begin button to display alert
```

Comment:
I think this should be the failing sign-up and in the now failing sign-up test should be the successful one.